_posts/2023-02-14-whatsnew-reporting-cli.markdown

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title:  "What’s new: OpenSearch Reporting CLI"
+authors:
+- maharup
+- jdbright
+- jadhanir
+- lijshu
+- alicejw
+date: 2023-02-14 00:00:00 -0700
+categories:
+ - technical-post
+ meta_keywords: "Reporting CLI, automate reports"
+ meta_description: "The Reporting CLI tool makes it convenient to generate reports externally without logging into OpenSearch Dashboards."
+---
+
+The OpenSearch project is happy to announce the release of the Reporting CLI tool that makes it convenient to generate reports externally without logging into OpenSearch Dashboards. This gives you the flexibility to programmatically generate reports and connect them with your preferred messaging system. You can now schedule and transport reports efficiently through email.
+
+In this post we’ll show you a few examples of how to leverage the Reporting CLI capabilities, such as:  
+
+* Generating a PDF from an eCommerce Dashboard that shows data visualizations and scheduling the report to run at a specific time using cron
```

Comment:
Thanks, i committed your suggestion. I think I missed putting the Dashboard (UI element) in bold instead of just uppercase. Should we change that, or is it ok to refer to it abstractly in lowercase. Thanks

![Dashboard  is a UI element in uppercase](https://user-images.githubusercontent.com/88908598/218177898-43695367-4778-420d-aac6-be7c83a5c083.png)

Reply:
If we're just referring to a generic dashboard, then it should remain lowercase. If we're talking about a specific, named dashboard and no other, then it looks like it would be "from the eCommerce Revenue Dashboard."